src/module_wrap.cc

```diff
@@ -991,12 +993,12 @@ Maybe<URL> ResolveExportsTarget(Environment* env,
       Utf8Value key_utf8(env->isolate(),
                          key->ToString(context).ToLocalChecked());
       std::string key_str(*key_utf8, key_utf8.length());
-      if (key_str == "node" || key_str == "import") {
+      if (conditions->Has(context, key).FromJust()) {
```

Comment:
You need to bubble exceptions up to JS rather than using `.FromJust()` for operations on JS sets (because these *can* fail in the presence of termination exceptions as they may use V8-internal JS code – and of course it’s also good practice to properly handle exceptions)

Reply:
Updated to do a `To(&bool)` and return an empty result. Afaict that should be enough to preserve the proper exception bubbling?